src/requests/request.rs

```diff
@@ -23,19 +23,20 @@ pub trait Request: HasPayload {
      * use it before it's integrated in async/await
      */
 
-    /// Type of error that may happen during sending the request to telegram.
+    /// A type of an error that may happen while sending a request to Telegram.
     type Err: std::error::Error + Send;
 
-    /// Type of future returned by [`send`](Request::send) method.
+    /// A type of the future returned by the [`send`](Request::send) method.
     type Send: Future<Output = Result<Output<Self>, Self::Err>> + Send;
 
-    /// Type of future returned by [`send_ref`](Request::send_ref) method.
+    /// A type of the future returned by the [`send_ref`](Request::send_ref)
+    /// method.
     ///
-    /// NOTE: it intentionally forbids borrowing from self
-    // though anyway we couldn't allow borrowing without GATs :sob:
+    /// Note: it intentionally forbids borrowing from `self` though anyway we
+    /// couldn't allow borrowing without GATs.
```

Comment:
I don't think we need to include this minor note in docs. But if we do, it's better to place it in braces or in a different sentence. IMHO.

Reply:
Yes, I'll just make this comment private.